package.json

```diff
@@ -194,6 +196,46 @@
     "./ja": "./dist/ja-answers-umd.js",
     "./ja/modern": "./dist/ja-answers-modern.js",
     "./ja/modern.min": "./dist/ja-answers-modern.min.js",
-    "./ja/template": "./dist/ja-answerstemplates.compiled.min.js"
+    "./ja/template": "./dist/ja-answerstemplates.compiled.min.js",
```

Comment:
I think we may need to issue a hotfix with these new entry points. The current SDK version published to NPM is supposed to have support for all these languages. But, without these entry points, the new language bundles are inaccessible. 

Can we have this change in a separate PR then, pointing to a hotfix branch?